### ISSUE TITLE
Errors detected by yamllint

### ISSUE THREAD

**Issue by rartych** [camaraproject/ConnectivityInsights#62]
**Problem description**
```
code/API_definitions/CAMARA Connectivity Insights API.yaml
  Error: 2:5 [indentation] wrong indentation: expected 0 but found 4
  Error: 7:8 [indentation] wrong indentation: expected 8 but found 7
  Error: 9:10 [indentation] wrong indentation: expected 8 but found 9
  Error: 68:5 [indentation] wrong indentation: expected 6 but found 4
  Error: 78:5 [indentation] wrong indentation: expected 6 but found 4
  Error: 88:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 89:13 [indentation] wrong indentation: expected 14 but found 12
  Error: 91:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 107:23 [indentation] wrong indentation: expected 20 but found 22
  Error: 213:12 [indentation] wrong indentation: expected 12 but found 11
  Error: 362:15 [indentation] wrong indentation: expected 16 but found 14
  Error: 446:12 [indentation] wrong indentation: expected 10 but found 11
  Error: 448:13 [indentation] wrong indentation: expected 13 but found 12
  Error: 457:16 [indentation] wrong indentation: expected 16 but found 15
  Error: 686:16 [indentation] wrong indentation: expected 16 but found 15
  Error: 735:16 [indentation] wrong indentation: expected 14 but found 15
  Error: 737:16 [indentation] wrong indentation: expected 14 but found 15
  Error: 739:16 [indentation] wrong indentation: expected 14 but found 15
  Error: 741:16 [indentation] wrong indentation: expected 14 but found 15
  Error: 743:16 [indentation] wrong indentation: expected 14 but found 15
  Error: 745:16 [indentation] wrong indentation: expected 14 but found 15
  Error: 770:15 [indentation] wrong indentation: expected 16 but found 14
  Error: 819:17 [indentation] wrong indentation: expected 14 but found 16

code/API_definitions/application-profiles.yaml
  Error: 2:5 [indentation] wrong indentation: expected 0 but found 4
  Error: 7:8 [indentation] wrong indentation: expected 8 but found 7
  Error: 9:10 [indentation] wrong indentation: expected 8 but found 9
  Error: 14:5 [indentation] wrong indentation: expected 6 but found 4
  Error: 24:5 [indentation] wrong indentation: expected 6 but found 4
  Error: 31:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 32:13 [indentation] wrong indentation: expected 14 but found 12
  Error: 34:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 51:23 [indentation] wrong indentation: expected 20 but found 22
  Error: 70:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 71:13 [indentation] wrong indentation: expected 14 but found 12
  Error: 73:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 99:23 [indentation] wrong indentation: expected 20 but found 22
  Error: 117:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 118:13 [indentation] wrong indentation: expected 14 but found 12
  Error: 120:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 139:23 [indentation] wrong indentation: expected 20 but found 22
  Error: 157:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 158:13 [indentation] wrong indentation: expected 14 but found 12
  Error: 160:11 [indentation] wrong indentation: expected 12 but found 10
  Error: 298:10 [indentation] wrong indentation: expected 10 but found 9
  Error: 300:11 [indentation] wrong indentation: expected 11 but found 10
  Error: 301:14 [indentation] wrong indentation: expected 12 but found 13
  Error: 304:14 [indentation] wrong indentation: expected 12 but found 13
  Error: 307:10 [indentation] wrong indentation: expected 10 but found 9
  Error: 418:47 [new-line-at-end-of-file] no new line character at the end of file
```

**Expected behavior**
Remove "3 dash: --- " in line 1 and correct indentation errors.
The suggested indentation values may change after removing dashes.
It is advised to use editor tool to correct indentation and apply the most popular indentation step of 2.

**Alternative solution**

**Additional context**

**Comment by Kevsy** [comment]
Thanks @rartych - also lots of spectral errors so I'll fix those at the same time.

**Comment by Kevsy** [comment]
Fixed in #64
